store/tikv/rawkv_test.go

```diff
@@ -126,6 +132,33 @@ func (s *testRawKVSuite) TestSimple(c *C) {
 	c.Assert(err, NotNil)
 }
 
+func (s *testRawKVSuite) TestBatchPut(c *C) {
+	testNum := 0
+	testKeys := make([][]byte, 0)
+	testValues := make([][]byte, 0)
+	countOfBatch := 0
+	size := 0
+	for i := 0; countOfBatch < 4; i++ {
```

Comment:
how about `for i := 0; size/rawBatchPutSize < 4; i++`?

Reply:
ok